spec/system/token_based_resume_controller_spec.rb

```diff
@@ -188,6 +188,8 @@
       page.driver.post "/sponsor-a-child/resume-application", params
 
       expect(page).to have_content(task_list_content)
+      visit "/sponsor-a-child/steps/10"
```

Comment:
Hiya - as discussed, could use (the brand new) helpers here?
```suggestion
      uam_click_task_list_link("Name")
```